testdata/e2e/release/init/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 1.1.0
```

Comment:
I need your help in this question : http://docs/document/d/1ctdcJyw8m9ZQbJIdUbO1ZOY2h68f5LoOpBteMJaI2GE/edit?resourcekey=0-xxJDV4GegDMXqzzMpBn_oQ&disco=AAABryh0U_o